pipeline-maven/src/test/resources/org/jenkinsci/plugins/pipeline/maven/test/test_maven_projects/maven_jar_with_jacoco_project/pom.xml

```diff
@@ -21,14 +21,14 @@
       <plugins>
         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.11.0</version>
+          <version>3.14.0</version>
```

Comment:
I just wonder if we could use some interpolation with properties from parent pom to avoid hardcoding and easy to forget upgrade :) 
```
<version>${maven-compiler-plugin.version}</version>
```
from 
https://github.com/jenkinsci/plugin-pom/blob/ba8384f207ef79a19a457c99da753b9333f301a3/pom.xml#L92
or
https://github.com/jenkinsci/plugin-pom/blob/ba8384f207ef79a19a457c99da753b9333f301a3/pom.xml#L81